canonical/canonical.com
# Add redirect from identity to diversity

## Done

Add redirect from identity to diversity

## QA

- Open the demo
- Go to /careers/diversity/identity
- See that it redirects to /careers/company-culture/diversity
